compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -431,8 +431,8 @@ impl<'tcx> interpret::Machine<'tcx> for CompileTimeMachine<'tcx> {
             // sensitive check here. But we can at least rule out functions that are not const at
             // all. That said, we have to allow calling functions inside a trait marked with
             // #[const_trait]. These *are* const-checked!
-            // FIXME: why does `is_const_fn_raw` not classify them as const?
-            if (!ecx.tcx.is_const_fn_raw(def) && !ecx.tcx.is_const_default_method(def))
+            // FIXME: why does `is_const_fn` not classify them as const?
```

Comment:
Can you change this to FIXME(effects) so we can grep for them?

```suggestion
            // FIXME(effects): why does `is_const_fn` not classify them as const?
```